org.nongnu.gsequencer.gsequencer.json

```diff
@@ -0,0 +1,241 @@
+	    "buildsystem": "simple",
+	    "build-commands": [
+		"./waf configure --prefix=/app --htmldir=/app/share/doc/jack/ --freebob=no --classic",
+		"./waf build -j $FLATPAK_BUILDER_N_JOBS",
+		"./waf install"
+	    ],
+	    "cleanup": [
+		"/bin",
+		"/share",
+		"/lib/jack",
+		"/lib/libjackserver.so*"
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/jackaudio/jack2/releases/download/v1.9.12/jack2-1.9.12.tar.gz",
+		    "sha256": "deefe2f936dc32f59ad3cef7e37276c2035ef8a024ca92118f35c9a292272e33"
+		}
+	    ]
+	},
+	
+	{
+	    "name": "liblo",
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "http://download.sf.net/sourceforge/liblo/liblo-0.29.tar.gz",
+		    "sha256": "ace1b4e234091425c150261d1ca7070cece48ee3c228a5612d048116d864c06a"
+		}
+	    ]
+	},      
+	{
+	    "name": "dssi",
+	    "rm-configure": true,
+	    "config-opts": [
+		"--disable-static"
+	    ],
+	    "cleanup": [
+		"/bin"
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "http://download.sf.net/sourceforge/dssi/dssi-1.1.1.tar.gz",
+		    "sha256": "f2c82b073a947c8255284249097667f9b14e660bf86186f3fcd3b3b3e087814e"
+		},
+		{
+		    "type": "script",
+		    "commands": [
+			"autoreconf -fiv"
+		    ],
+		    "dest-filename": "autogen.sh"
+		}
+	    ]
+	},
+
+	{
+	    "name": "audiofile",
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "http://audiofile.68k.org/audiofile-0.3.6.tar.gz",
+		    "sha256": "cdc60df19ab08bfe55344395739bb08f50fc15c92da3962fac334d3bff116965"
+		},
+		{
+		    "type": "patch",
+		    "path": "patches/audiofile-gcc6.patch"
+		}
+	    ]
+	},
+	
+	{
+	    "name": "libinstpatch",
+	    "rm-configure": true,
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://downloads.sourceforge.net/project/swami/libinstpatch/1.0.0/libinstpatch-1.0.0.tar.gz",
+		    "sha256": "8943e2226d97609104365f0105be87b326a0b2335cea22d29604a45f8ef1cee7"
+		},
+		{
+		    "type": "patch",
+		    "strip-components": 0,
+		    "path": "patches/instpatch.examples.patch"
+		},
+		{
+		    "type": "script",
+		    "commands": [
+			"autoreconf -fiv"
+		    ],
+		    "dest-filename": "autogen.sh"
+		}
+	    ]
+	},
+	
+	{
+	    "name": "lv2",
+	    "buildsystem": "simple",
+	    "build-commands": [
+		"./waf configure --prefix=/app --docdir=/app/share/doc/lv2",
+		"./waf build -j $FLATPAK_BUILDER_N_JOBS",
+		"./waf install"
+	    ],
+	    "cleanup": [
+		"/bin",
+		"/share",
+		"/lib/jack",
+		"/lib/libjackserver.so*"
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "http://lv2plug.in/spec/lv2-1.16.0.tar.bz2",
+		    "sha256": "dec3727d7bd34a413a344a820678848e7f657b5c6019a0571c61df76d7bdf1de"
+		}
+	    ]
+	},
+
+	{
+	    "name": "gsequencer",
+	    "config-opts": [
+		"BUILDCC=/usr/bin/gcc -std=gnu99"
+	    ],
+	    "post-install": [
+		"mv /app/bin/gsequencer /app/bin/gsequencer.bin",
+		"mv gsequencer.sh /app/bin/gsequencer",
```

Comment:
Why does this not work upstream? Why are you doing workarounds here?

Reply:
GSequencer supports environment variables for splash screen and such. But not used here.

Reply:
I'm not sure why the flatpak should have a different launcher than other packages is my point.

Reply:
VLC does the same.

Reply:
Whataboutism isn't helpful frankly.

This is a weird thing to do, flatpaks should contain normal upstream releases. Since you are upstream you control this and downstream changes here don't make sense. The VLC package isn't directly supported by upstream.

Reply:
I just run the build with the new changes.